Towny/src/main/java/com/palmergames/bukkit/towny/tasks/TeleportWarmupTimerTask.java

```diff
@@ -64,8 +67,10 @@ public void run() {
 				// Only teleport & add cooldown if player is valid
 				if (player == null)
 					continue;
-				
-				PaperLib.teleportAsync(player, request.destinationLocation(), TeleportCause.COMMAND);
+
+				List<Entity> pets = SpawnUtil.getPets(player);
+				pets.add(player);

```

Comment:
Would be better to call it entities since the player is added to it (and is not a pet)

Reply:
Came here to say this.